src/components/button-bar/button-bar.test.js

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable react/prop-types */
+import * as React from "react";
+import ButtonBar from ".";
+import Button from "../button";
+import {
+  BUTTON_BAR_SIZES,
+  BUTTON_BAR_ICON_POSITIONS,
+} from "./button-bar.config";
+
+import { buttonDataComponent } from "../../../cypress/locators/button";
+
+import { icon } from "../../../cypress/locators";
+import { positionOfElement, keyCode } from "../../../cypress/support/helper";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+const ButtonBarCustom = ({ onClick, ...props }) => {
+  return (
+    <ButtonBar onClick={onClick} {...props} ml={2} mt={2}>
+      <Button>Button 1</Button>
+      <Button>Button 2</Button>
+      <Button>Button 3</Button>
+    </ButtonBar>
+  );
+};
+
+const ButtonBarIconPosition = ({ iconPositionProp, ...props }) => {
+  return (
+    <ButtonBar {...props} size={BUTTON_BAR_SIZES[0]} ml={2} mt={2}>
```

Comment:
You don't need to set `size` here as it has a default value (`medium`) but it's not a big deal